lib/screens/form.dart

```diff
@@ -0,0 +1,285 @@
+                        thumbShape:
+                            RoundSliderThumbShape(enabledThumbRadius: 12.0),
+                        thumbColor: Colors.white,
+                        overlayColor: Colors.blue.withAlpha(32),
+                        overlayShape:
+                            RoundSliderOverlayShape(overlayRadius: 28.0),
+                        tickMarkShape: RoundSliderTickMarkShape(),
+                        activeTickMarkColor: Colors.lightBlue[400],
+                        inactiveTickMarkColor: Colors.white,
+                        valueIndicatorShape: PaddleSliderValueIndicatorShape(),
+                        valueIndicatorColor: Colors.white,
+                        valueIndicatorTextStyle: TextStyle(
+                          color: Colors.blue[300],
+                          fontWeight: FontWeight.bold,
+                          fontSize: 17,
+                        ),
+                      ),
+                      child: Slider(
+                        value: sliderValue,
+                        min: 0,
+                        max: 8,
+                        divisions: 8,
+                        label: myLetterGrade,
+                        onChanged: (value) {
+                          setState(
+                            () {
+                              sliderValue = value;
+                              if (sliderValue < 2.0) {
+                                myLetterGrade = "F";
+                                _grade = 0.0;
+                              }
+                              if (sliderValue >= 2.0 && sliderValue < 4.0) {
+                                myLetterGrade = "D";
+                                _grade = 1.0;
+                              }
+                              if (sliderValue >= 4.0 && sliderValue < 5.0) {
+                                myLetterGrade = "C";
+                                _grade = 2.0;
+                              }
+                              if (sliderValue >= 5.0 && sliderValue < 6.0) {
+                                myLetterGrade = "C+";
+                                _grade = 2.5;
+                              }
+                              if (sliderValue >= 6.0 && sliderValue < 7.0) {
+                                myLetterGrade = "B";
+                                _grade = 3.0;
+                              }
+                              if (sliderValue >= 7.0 && sliderValue < 8.0) {
+                                myLetterGrade = "B+";
+                                _grade = 3.5;
+                              }
+                              if (sliderValue == 8.0) {
+                                myLetterGrade = "A";
+                                _grade = 4.0;
+                              }
+                            },
+                          );
+                        },
+                      ),
+                    ),
+                  ),
+                  Align(
```

Comment:
No need to for align if you'll use container...container has a align attribute already